pico/api/src/main/java/io/helidon/pico/api/Interceptor.java

```diff
@@ -32,6 +32,7 @@ public interface Interceptor {
      * @param args  the arguments to the call
      * @param <V>   the return value type (or {@link Void} for void method elements)
      * @return the return value to the caller
+     * @throws InvocationException if there is an errors during invocation chain processing
```

Comment:
either `is an error` or `are errors` - typo

Reply:
pushed